clang/lib/AST/ASTImporter.cpp

```diff
@@ -3008,8 +3008,22 @@ ExpectedDecl ASTNodeImporter::VisitRecordDecl(RecordDecl *D) {
         // declaration), ASTContext does not check it.
         SmallVector<Decl *, 2> Redecls =
             getCanonicalForwardRedeclChain(D2CXX);
-        const Type *FrontTy =
-            cast<CXXRecordDecl>(Redecls.front())->getTypeForDecl();
+
+        // FIXME: This is an LLDB-specific fix for crashes
+        // related to rdar://120862220, where the redeclaration
```

Comment:
We usually don't put rdar links into the code since they're opaque to most. Maybe move it to the commit message?